platforms/hyperledger-besu/configuration/roles/create/helm_component/templates/validator.tpl

```diff
@@ -69,6 +69,9 @@ spec:
         rpc: {{ peer.rpc.port }}
         ws: {{ peer.ws.port }}
 
+    permissioning:
```

Comment:
This should also logically go within the node section itself

Reply:
updated